MvvmCross/iOS/iOS/Views/Presenters/Attributes/IMvxOverridePresentationAttribute.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System;
+namespace MvvmCross.iOS.Views.Presenters.Attributes
+{
+    public interface IMvxOverridePresentationAttribute
+    {
+        MvxBasePresentationAttribute OverridePresentationAttribute();
```

Comment:
No this is not possible, the different iOS presentation attributes do not implement the `IMvxPresentationAttribute` (this interface also no longer exists, unless my search skills are lacking).